packages/components/src/Columns/Columns.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Children } from 'react';
+import { useStyles } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-keyed-flatten-children';
+
+type ColumnsProps = {
+  className?: string;
+  space?: number;
+  horizontalAlign?: 'left' | 'right' | 'center';
+  verticalAlign?: 'top' | 'bottom' | 'center';
+  title?: string; // Should only be used for testing.
+};
+
+export const Columns: React.FC<ColumnsProps> = ({
+  space = 0,
+  horizontalAlign = 'left',
+  verticalAlign = 'top',
+  className,
+  children,
+  ...props
+}) => {
+  let columnItems = flattenChildren(children);
+  let childClassNames = useStyles({ p: `${space / 2}px` });
+
+  // horizontal Alignment
+  let justify = 'flex-start';
+  if (horizontalAlign === 'right') {
+    justify = 'flex-end';
+  } else if (horizontalAlign === 'center') {
+    justify = 'center';
+  }
+
+  // vertical Alignment
+  let alignItems = 'flex-start';
+  if (verticalAlign === 'bottom') {
+    alignItems = 'flex-end';
+  } else if (verticalAlign === 'center') {
+    alignItems = 'center';
+  }
+
+  return (
+    <Box p={space} display="flex" className={className}>
+      <Box
+        width={`calc(100% + ${space}px)`}
+        m={-space / 2}
+        display="flex"
+        flexWrap="wrap"
+        alignItems={alignItems}
+        justifyContent={justify}
+        {...props}
+      >
+        {Children.map(
```

Comment:
Use flatten children?

Reply:
Used in in line 22. In Stack component and in Braid DS is it also used like this.